src/main/java/frc/robot/RobotContainer.java

```diff
@@ -28,15 +36,22 @@ public RobotContainer() {
 
     private void configureBindings() {
 
-        //* */ weirdo ripple effect
+        //* weirdo ripple effect */
         new JoystickButton(joystick, 3)
         .toggleOnTrue(new InstantCommand(() -> m_led.setRGB(0, 0, 255))
         .andThen(new RepeatCommand(Commands.sequence(new InstantCommand(() -> m_led.DecreaseAllLedsBrightness()),
         new WaitCommand(0),
         new LighNextLed(m_led)))));
 
+        //* blinking effect */
         new JoystickButton(joystick, 2)
-        .toggleOnTrue(Commands.repeatingSequence(m_led.AllLEDSBlinking(255, 255, 0, 0.3)));
+            .toggleOnTrue(new SelectCommand(
+                Map.ofEntries(
+                Map.entry(LEDColorState.RED,Commands.repeatingSequence(m_led.AllLEDSBlinking(255, 0, 0, 0.3))),
+                Map.entry(LEDColorState.GREEN,Commands.repeatingSequence(m_led.AllLEDSBlinking(0, 255, 0, 0.3))),
+                Map.entry(LEDColorState.BLUE,Commands.repeatingSequence(m_led.AllLEDSBlinking(0, 0, 255, 0.3)))
+                ), () -> ledColorState()));

```

Comment:
bu fonksiyon white döndürüyor ama map'te white için bir entry yok o yüzden çalışmamıştır